generate_readme/data.yaml

```diff
@@ -26,9 +31,31 @@ items:
   repo: https://github.com/mastodon/mastodon
   liam_erd_web_url: https://liambx.com/erd/p/github.com/mastodon/mastodon/blob/main/db/schema.rb?showMode=ALL_FIELDS
   app_description: Your self-hosted, globally interconnected microblogging community.
-  erd_description: |
-    This project features a comprehensive schema of tables for managing users, statuses, and media attachments. It accommodates multi-tenant data through tables for users, accounts, templates, and submissions. The schema ensures data integrity through foreign keys and indexes, while providing dedicated support for event tracking, OAuth, file attachments, and webhooks. This demonstrates a robust, form-driven application flow.
+  schema_technical_terms:
+  - "99 tables"
+  - "PostgreSQL"
+  - "Rails"
+  - "100+ FK"
+  - "bigint"
+  feature_description: "Account Management, Timelines, Notifications, Conversations, Relationships, Content Filtering, Distributed Federation"
   headline_genres:
   - "Communication - Social Networks and Forums"
   table_count: 99
   checked_at: '2025-02-12'
+- title: "Maybe"
+  repo: "https://github.com/maybe-finance/maybe"
+  liam_erd_web_url: "https://liambx.com/erd/p/github.com/maybe-finance/maybe/blob/main/db/schema.rb"
+  app_description: "The OS for personal finance"
+  schema_technical_terms:
+    - "52 tables"
+    - "PostgreSQL"
+    - "Rails"
+    - "Multi-Tenancy"
+    - "Async Jobs"
+    - "uuid"
+    - "jsonb"
+  feature_description: "Bank Aggregation, Budgeting, Investments, Payment Tracking, Multi-Tenancy, Data Enrichment, Data Syncing"
+  headline_genres:
+    - "Money, Budgeting & Management"
+  table_count: 52
```

Comment:
The ``table_count`` is not in ``generate.rb``, but I personally feel it is necessary. 🤔💭 It would be less duplicative to remove it from ``schema_technical_terms`` instead.

Reply:
Ooooh, you're right! I'll fix it so that table_count is used first!

Reply:
I fix it: https://github.com/liam-hq/awesome-er-diagrams/commit/9befea45d34c27287e90b8b31a6582f06951a162

Reply:
Thank you so much!💫